packages/manager/apps/web/client/app/domain/dashboard/domain.service.js

```diff
@@ -135,6 +135,12 @@ angular.module('services').service(
       });
     }
 
+    getDomainsService(serviceName) {
+      return this.OvhHttp.get(`/domain/${serviceName}`, {
```

Comment:
you could use `$http.get` instead.
```suggestion
      return this.$http.get(`/domain/${serviceName}`).then(({data}) => data)
```

Reply:
we use in this file OvhHttp

getTasks(serviceName, params = {}) {
      return this.OvhHttp.get(`/domain/${serviceName}/task`, {
      return this.OvhHttp.get(`/domain/${serviceName}/task`, {
        rootPath: 'apiv6',
        rootPath: 'apiv6',
        params,
        params,
      });
      });
    }
    }
    /**
    /**
     * Get tasks by status
     * Get tasks by status
     * @param {string} serviceName
     * @param {string} serviceName
     * @param {string} fn
     * @param {string} fn
     * @param {Array} status
     * @param {Array} status
     */
     */
    getTasksByStatus(serviceName, fn, status = []) {
    getTasksByStatus(serviceName, fn, status = []) {
      const promisesTasks = map(status, (st) =>
      const promisesTasks = map(status, (st) =>
        this.getTasks(serviceName, { status: st, function: fn }),
        this.getTasks(serviceName, { status: st, function: fn }),
      );
      );
      return this.$q.all(promisesTasks).then(flatten);
      return this.$q.all(promisesTasks).then(flatten);
    }
    }
    /**
    /**
     * Get task
     * Get task
     * @param {string} serviceName
     * @param {string} serviceName
     * @param {string} id
     * @param {string} id
     */
     */
    getTask(serviceName, id) {
    getTask(serviceName, id) {
      return this.OvhHttp.get(`/domain/${serviceName}/task/${id}`, {
      return this.OvhHttp.get(`/domain/${serviceName}/task/${id}`, {
        rootPath: 'apiv6',
        rootPath: 'apiv6',
      }).then((originalTasksDetails) => {
      }).then((originalTasksDetails) => {
        const tasksDetails = clone(originalTasksDetails);
        const tasksDetails = clone(originalTasksDetails);
        tasksDetails.status =
        tasksDetails.status =
          isString(tasksDetails.status) && tasksDetails.status.toUpperCase();
          isString(tasksDetails.status) && tasksDetails.status.toUpperCase();
        return tasksDetails;
        return tasksDetails;
      });
      });
    }
    }